doc/rados/operations/health-checks.rst

```diff
@@ -1218,96 +1221,96 @@ For more information, see :ref:`associate-pool-to-application`.
 ______________
 
-One or more pools is approaching a configured fullness threshold.
+One or more pools are approaching a configured fullness threshold.
 
-One threshold that can trigger this warning condition is the
-``mon_pool_quota_warn_threshold`` configuration option.
+One of the several thresholds that can raise this health check is determined by
+the ``mon_pool_quota_warn_threshold`` configuration option.
 
-Pool quotas can be adjusted up or down (or removed) with:
+Pool quotas can be adjusted up or down (or removed) by running the following
+commands:
 
 .. prompt:: bash $
 
    ceph osd pool set-quota <pool> max_bytes <bytes>
    ceph osd pool set-quota <pool> max_objects <objects>
 
-Setting the quota value to 0 will disable the quota.
+To disable a quota, set the quota value to 0.
 
-Other thresholds that can trigger the above two warning conditions are
-``mon_osd_nearfull_ratio`` and ``mon_osd_full_ratio``.  Visit the
-:ref:`storage-capacity` and :ref:`no-free-drive-space` documents for details
-and resolution.
+Other thresholds that can raise the two health checks above are
+``mon_osd_nearfull_ratio`` and ``mon_osd_full_ratio``. For details and
+resolution, see :ref:`storage-capacity` and :ref:`no-free-drive-space`.
 
 OBJECT_MISPLACED
 ________________
 
-One or more objects in the cluster is not stored on the node the
-cluster would like it to be stored on.  This is an indication that
-data migration due to some recent cluster change has not yet completed.
+One or more objects in the cluster are not stored on the node that the cluster
+would prefer that they be stored on. This alert is an indication that data
+migration due to a recent cluster change has not yet completed.
 
-Misplaced data is not a dangerous condition in and of itself; data
-consistency is never at risk, and old copies of objects are never
-removed until the desired number of new copies (in the desired
-locations) are present.
+Misplaced data is not a dangerous condition in and of itself; data consistency
+is never at risk, and old copies of objects will not be removed until the
+desired number of new copies (in the desired locations) has been created.
 
 OBJECT_UNFOUND
 ______________
 
-One or more objects in the cluster cannot be found.  Specifically, the
-OSDs know that a new or updated copy of an object should exist, but a
-copy of that version of the object has not been found on OSDs that are
-currently online.
+One or more objects in the cluster cannot be found. More precisely, the OSDs
+know that a new or updated copy of an object should exist, but no such copy has
+been found on OSDs that are currently online.
 
 Read or write requests to unfound objects will block.
 
-Ideally, a down OSD can be brought back online that has the more
-recent copy of the unfound object.  Candidate OSDs can be identified from the
-peering state for the PG(s) responsible for the unfound object:
+Ideally, a "down" OSD that has a more recent copy of the unfound object can be
```

Comment:
`down` is a literal state, suggest backticks instead of quotes.  Also suggest the same for `unfound`; evaluate the two separately.

Reply:
unfinished business -- use/mention matters will be clarified later.